### ISSUE TITLE
[BUG] VO tools doesn't read the current progress of a Kirby progress circle

### ISSUE THREAD

**Issue by tfl-jyskebank-dk** [kirbydesign/designsystem#3796]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When using a Voice Over tool it should read the current progress for the progress circle. 

## Describe how to reproduce the bug
Use a Voice Over tool on a progress circle with a progress set. 

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
That the Voice Over tools would read the specified progress. 

## Add any screenshots
https://github.com/user-attachments/assets/83acf619-e182-4b2e-8f60-8425040da816

## Please complete the following information:
- OS: iOS, macOS
- Browser Safari
- Browser version: iOS 18.3 Safari,  MacOS safari 18.2

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
